### DISCUSSION TITLE
error on v3.59: the procedure entry point discard virtual memory could not be located in the dynamic link library kernek32.dll

### DISCUSSION THREAD

**Discussion by q12a** [mifi/lossless-cut#1890] [General]
error on v3.59: the procedure entry point discard virtual memory could not be located in the dynamic link library kernek32.dll
![Screenshot_1](https://github.com/mifi/lossless-cut/assets/71429061/91c8803a-ae96-488e-90fd-2bf74daed1ad)

**Comment by mifi** [comment]
https://github.com/mifi/lossless-cut/blob/master/issues.md

**Comment by q12a** [comment]
Yes, I use w7 and your FAQ page sais this indeed:
_If you receive a KERNEL32.dll error, it's probably because you're running Windows 7, 8 or 8.1 which are https://github.com/mifi/lossless-cut/discussions/1476_
- But, why dont you make it work for w7? w10 and w11 are full of NEW errors and bugs, while w7 is a consecrated and tested in time OS. You should integrate it as well, if not all the time, at least from time to time. Like special versions only for w7 !
Thank you.

**Reply by mifi** [reply]
You can see discussions in #1476 where I explain why support was dropped. You can still use old versions on windows 7 or fork it yourself to maintain w7 support if you want to
